src/main/java/com/coveros/selenified/Capabilities.java

```diff
@@ -267,8 +267,16 @@ private Boolean runHeadless() {
      * @param extraCapabilities any additional parameters to set for selenium
      */
     public void addExtraCapabilities(DesiredCapabilities extraCapabilities) {
+        setDesiredCapabilities();
         if (extraCapabilities != null && browser.getName() != BrowserName.NONE) {
             desiredCapabilities = desiredCapabilities.merge(extraCapabilities);
+            // look for specifics
+            if(!extraCapabilities.isJavascriptEnabled()) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "isJavascriptEnabled"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)